Alignment/CommonAlignment/python/tools/trackselectionRefitting.py

```diff
@@ -198,8 +213,10 @@ def getSequence(process, collection,
                 "minimumHits": 10,
                 })
     else:
-        print "Unknown input track collection:", collection
-        sys.exit(1)
+        raise ValueError("Unknown input track collection: {}".format(collection))
+
+    if cosmicTrackSplitting and not isCosmics:
+        raise ValueError("Can only do cosmic track splitting for cosmics.")
```

Comment:
is `ValueError` the correct type of exception?
sounds more like a logic error, which is likely best described as `RuntimeError`